src/lua/bpf/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2016, Marek Vavrusa <marek@vavrusa.com>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
```

Comment:
all exported header files and examples have to have it. Single line like in tools/*.py would be enough though. Ideally all files... Just to avoid any ambiguity.
Pls squash and force push to have single commit for clean history.
Thanks!